src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -173,6 +180,17 @@ public void resendRPANotifications(
         resendClaimsToRPA(existingClaims, authorisation);
     }
 
+    @PostMapping(value = "/sendMediation", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ApiOperation("Generate and Send Mediation Report for Telephone Mediation Service")
+    public void sendMediation(
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation,
+        @RequestBody MediationRequest mediationRequest
+    ) {
```

Comment:
There will be validation that it _is_ a `LocalDate`. We can request a report of mediation agreements established in the future and always get an empty result. I don't see much value in blocking it as a special case.

Reply:
I feel that it's always better to provide feedback 1. whenever the user does something wrong and 2. this endpoint is for support trying to re-send a mediation file and I'm only guessing here .. that they're expecting something to be sent. So they're not getting any feedback. But ... its fairly minor and more for a BA to comment on rather than technical. Not going to reject this PR if you don't, so feel free to to resolve this if you're sure it's not necessary.

Reply:
If exceptions are thrown by our requests we will see errors returned to our consoles, no bother! Debugging is a constant problem in CMC due to lack of documentation and logging but at this point we don't know what we might need to see so I'll just leave it as the defaults for now.